### PR TITLE
[NT-459] Qualtrics Surveys

### PR DESCRIPTION
**Pull request by justinswart**
# 📲 What

Adds Qualtrics Surveys.

Includes the work from #987, #992 and #1001.

# 🤔 Why

We plan to run some surveys to gather feedback from our users to help us to improve the app.

# 🛠 How

- Added the SDK in #987.
- Added presentation of surveys via local notification in #992.
- Added some custom properties to help us filter in #1001.

# 👀 See

### CODE REVIEW

**Comment by nativeksr** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Big PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: Cartfile [hunk 1]
```diff
@@ -13,3 +13,4 @@ github "stripe/stripe-ios" == 18.0.0
 github "uber/ios-snapshot-test-case" == 4.0.0
 github "thoughtbot/Argo" "39f06f089d25c111444e5a85eef64586e54756ac"
 github "thoughtbot/Curry" "b6bf27ec9d711f607a8c7da9ca69ee9eaa201a22"
+binary "https://s3-us-west-2.amazonaws.com/si-mobile-sdks/ios/Qualtrics.json" == 1.10
```
Comment (ifbarrera) [inline Cartfile]: 😬

File: Kickstarter-iOS/AppDelegate.swift [hunk 2]
```diff
@@ -199,6 +200,36 @@ internal final class AppDelegate: UIResponder, UIApplicationDelegate {
       .observeForUI()
       .observeValues { [weak self] in self?.findRedirectUrl($0) }
 
+    self.viewModel.outputs.configureQualtrics
+      .observeValues { [weak self] config in
+        let properties = Properties()
```
Comment (ifbarrera) [inline Kickstarter-iOS/AppDelegate.swift]: Maybe we should break all his out into a function? Just a thought.

File: Cartfile (hunk 1 above)
Comment (justinswart) [reply Cartfile]: Yip 🤷‍♂ https://api.qualtrics.com/docs/getting-started-with-the-mobile-app-sdk-on-ios#section-installing-the-framework-with-carthage

**Pull request merged**